src/App.js

```diff
@@ -1,9 +1,10 @@
+import { PortfolioWrapper } from 'components/PortfolioWrapper';
```

Comment:
Good solution to keep the app.js clean by importing a wrapper with all the content, nice!